runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -525,4 +566,36 @@ public void onFocusChanged(FocusChanged focusChanged)
 			setHotKeyPressed(false);
 		}
 	}
+
+	public void notifyValuableItem(GroundItem item, ValueThreshold threshold)
+	{
+		final Player local = client.getLocalPlayer();
+		StringBuilder notificationString = new StringBuilder();
+
+		if (threshold == ValueThreshold.INSANE)
+		{
+			notificationString.append("an ");
+		}
+		else
+		{
+			notificationString.append("a ");
+		}
+
+		notificationString.append(threshold);
+
+		if (threshold != ValueThreshold.HIGHLIGHTED)
+		{
+			notificationString.append(" value");
+		}
+
+		notificationString.append(" drop: ")
+			.append(item.getName())
+			.append("");
+
+		if (item.getQuantity() > 1)
+		{
+			notificationString.append(" x ").append(item.getQuantity());
+		}
+		notifier.notify("[" + local.getName() + "] received " + notificationString.toString() + "!");
```

Comment:
empty line above

Reply:
I would also use here same format as overlay uses for displaying an item (and use some shared logic)